src/TestProblem/Hydro/Bondi/Flu_ResetByUser_Bondi.cpp

```diff
@@ -17,7 +17,7 @@ extern double Bondi_SinkMomZAbs;
 extern double Bondi_SinkEk;
 extern double Bondi_SinkEt;
 extern int    Bondi_SinkNCell;
-extern double Bondi_SinkMass_tot;
+       double SinkMass_OneSubStep_AllRank;
```

Comment:
This can be declared as a local variable within `Flu_ResetByUser_API_Bondi` (just like `SinkMass_OneSubStep_ThisRank`)